cmd/portable-package.rb

```diff
@@ -1,74 +1,74 @@
 # frozen_string_literal: true
 
-require "cli/parser"
+require "abstract_command"
 
 module Homebrew
-  def self.portable_package_args
-    Homebrew::CLI::Parser.new do
-      usage_banner <<~EOS
-        `portable-package` <formulae>
+  module Cmd
+    class PortablePackageCmd < AbstractCommand
+      cmd_args do
+        usage_banner <<~EOS
+          `portable-package` <formulae>
 
-        Build and package portable formulae.
-      EOS
-      switch "--no-uninstall-deps",
-             description: "Don't uninstall all dependencies of portable formulae before testing."
-      switch "-v", "--verbose",
-             description: "Pass `--verbose` to `brew` commands."
-      named_args :formula, min: 1
+          Build and package portable formulae.
+        EOS
+        switch "--no-uninstall-deps",
+              description: "Don't uninstall all dependencies of portable formulae before testing."
+        switch "-v", "--verbose",
+              description: "Pass `--verbose` to `brew` commands."
+        named_args :formula, min: 1
+      end
     end
```

Comment:
This is misplaced

Reply:
@Bo98 I've created this PRs en-masse. Can skip review here, I'm not going to merge until working.